Makefile

```diff
@@ -1,21 +1,72 @@
-default: build run 
+SHELL := /bin/bash
```

Comment:
Not all have `bash` in `/bin` (*cough* NixOS *cough*)

Use `which` or smth, `SHELL = $(shell which bash)`

Reply:
Also probably remove this. Remove dependency on shell.